projects/valtimo/process-link/src/lib/components/select-form/select-form.component.ts

```diff
@@ -110,10 +107,13 @@ export class SelectFormComponent implements OnInit, OnDestroy {
   }
 
   private updateProcessLink(): void {
-    this.stateService.selectedProcessLink$.pipe(take(1)).subscribe(selectedProcessLink => {
+    combineLatest([this.stateService.selectedProcessLink$, this.stateService.viewModelEnabled$])
+      .pipe(take(1))
+      .subscribe(([selectedProcessLink, viewModelEnabled]) => {
       const updateProcessLinkRequest: FormProcessLinkUpdateRequestDto = {
         id: selectedProcessLink.id,
         formDefinitionId: this._selectedFormDefinition.id,
+        viewModelEnabled: viewModelEnabled
```

Comment:
This has the same effect

```suggestion
        viewModelEnabled
```